doc/classes/RemoteTransform.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RemoteTransform" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		RemoteTransform mirrors the [Transform] of another Node in the scene.
 	</brief_description>
 	<description>
+		RemoteTransform mirrors the [Transform] of another Node (called the [code]remote node[/code]) in the scene.
+		It can be set to track another Node's position, rotation, scale or all of those and update its own accordingly, using either global or local coordinates.
+		The [code]remote node[/code] has to inherit from [Spatial] (i.e. it actually has to have a [Transform] to mirror).
```

Comment:
- why is "remote node" in [code]? It should either be text "remote node" or a link to the actual [member remote_path]
- may be redundant if you specify [Spatial] in the first sentence.